sources/packages/backend/apps/workers/src/controllers/application/_tests_/e2e/update-application-status.ts

```diff
@@ -0,0 +1,38 @@
+import { createFakeWorkerJob } from "../../../../../test/utils/worker-job-mock";
+import { IOutputVariables, ZeebeJob } from "zeebe-node";
+import { ApplicationStatus } from "@sims/sims-db";
+import {
+  ApplicationUpdateStatusJobHeaderDTO,
+  ApplicationUpdateStatusJobInDTO,
+} from "../../application.dto";
+
+/**
+ * Creates a fake update application status payload.
+ * @param applicationId application id.
+ * @param fromStatus from status.
+ * @param toStatus to status.
+ * @returns fake update application status payload.
+ */
+export function createFakeUpdateApplicationStatusPayload(
+  applicationId: number,
+  fromStatus: ApplicationStatus,
+  toStatus: ApplicationStatus,
+): Readonly<
+  ZeebeJob<
+    ApplicationUpdateStatusJobInDTO,
+    ApplicationUpdateStatusJobHeaderDTO,
+    IOutputVariables
+  >
+> {
+  const variables = {
+    applicationId,
+  };
+  return createFakeWorkerJob<
+    ApplicationUpdateStatusJobInDTO,
+    ApplicationUpdateStatusJobHeaderDTO,
+    IOutputVariables
+  >({
+    variables,
+    customHeaders: { fromStatus, toStatus },
```

Comment:
Since `customHeaders` is following this format I would prefer to have the variables doing the same.
```ts
variables: { applicationId },
customHeaders: { fromStatus, toStatus },
```